build/build.sh

```diff
@@ -17,6 +17,16 @@
 # This script builds the application from source for multiple platforms.
 set -e
 
+VERSION_FILE_PATH="$GOPATH/src/github.com/openebs/cstor-operators/VERSION"
```

Comment:
Based on the below comments, it will be good to delete this file. Also helps with cases where builds need to happen on forked repos.

Reply:
Yes, I have a PR #106 to fix